tests/cxx/inheriting_ctor.cc

```diff
@@ -0,0 +1,18 @@
+
+#include "inheriting_ctor-i1.h"
+
+//IWYU: Derived is defined in .*-i2.h
+void func() { Derived(1); }
```

Comment:
This would be easier to read if you broke it over multiple lines and named the `Derived` instance, something like:

    void func() {
      // IWYU ...
      Derived d(1);
    }